toontown/leveleditor/DNASerializer.py

```diff
@@ -0,0 +1,265 @@
+from panda3d.core import Filename
+import os
+from tkinter import *
+from tkinter.filedialog import *
+from direct.directnotify import DirectNotifyGlobal
+
+dnaDirectory = Filename.expandFrom(userfiles)
+
+class DNASerializer:
+    notify = DirectNotifyGlobal.directNotify.newCategory('LevelEditor')
+
+    def __init__(self, editor):
+        self.editor = editor
```

Comment:
We don't need this. We can use `base.le`.
You can also make these @staticmethods and call them via DNASerializer.loadDNAFromFile instead of self.serializer